entrypoint.sh

```diff
@@ -1,40 +1,46 @@
 #!/bin/bash -e
 
 AWS_REGION="${AWS_REGION:-us-east-1}"
-MISBEHAVING_NOTICE="may be misbehaving. In a perfect world, our monitoring detected this problem and Platform Engineering was alerted... but just in case, please let us know."
 
-if [ ${CONSUL_ADDR} ]
+if [ "${APP_ENV}" == "dev" ]
 then
-  if consul-template -consul-addr=$CONSUL_ADDR -template=/export-consul.ctmpl:/tmp/export-consul.sh -once -max-stale=0
+  source /dev-entrypoint.sh
```

Comment:
i think this should be just `/dev-entrypoint.sh` here - instead of `source /dev-entrypoint.sh` since you're executing a script - not setting env vars

Reply:
Its a subtle difference.  Source loads the code into this file, and then gets executed as part of the main script .. where as the alternative forks a new process and executes it there.

> `source` is a bash shell built-in command that executes the content of the file passed as argument, in the current shell.

Reply:
i did not know this. can it be added as a comment so the next person reading the script doesn't miss this info?

Reply:
I would feel really strangely about documenting how bash internals work in a comment here.

Reply:
@bellaweo do you have a suggestion on how this comment should be worded?

Reply:
something to the effect of "we use source here because we want dev-entrypoint.sh to be executed as part of this entrypoint.sh script and not as a new process"